h/static/styles/partials-v2/_api-token.scss

```diff
@@ -0,0 +1,14 @@
+.api-token {
+  border: 1px solid $grey-3;
+  border-radius: 3px;
+  display: block;
+  min-height: 50px;
+  width: 100%;
+
+  color: $grey-6;
+  text-align: center;
+}
+
+.api-token--text {
```

Comment:
The `--modifier` here would indicate a different version of `api-token`, perhaps to indicate a read only vs read/write token for example.

I think we probably want a `form-help-text` component, that can go in `_form.scss` and we can then use it for things like the hint text in https://trello.com/c/gGtgaRg9/637-form-field-hint-enhancement on touch devices.